worker/utils/utils.js

```diff
@@ -49,6 +51,27 @@ module.exports = {
     })
   },
 
+  async encryptJob(string1, string2){
```

Comment:
What are string1 and string2, and why is it important that they be concatenated with the password in this function?